addons/docs/src/blocks/Anchor.tsx

```diff
@@ -7,5 +7,7 @@ export interface AnchorProps {
 }
 
 export const Anchor: FC<AnchorProps> = ({ storyId, children }) => (
-  <div id={anchorBlockIdFromId(storyId)}>{children}</div>
+  <div id={anchorBlockIdFromId(storyId)} style={{ textAlign: 'center' }}>
```

Comment:
why text-align center?

Reply:
I added this so no matter what size the iFrame has, the component will always be centered on the Docs Page...

Reply:
Without it:
![image](https://user-images.githubusercontent.com/48736073/70853947-4a4eba80-1e72-11ea-8157-0fb19e828a97.png)

With it:
![image](https://user-images.githubusercontent.com/48736073/70853950-56d31300-1e72-11ea-8f18-93ddc1e97960.png)

Reply:
that visual outcome looks ok to me (I'm no designer), but this shouldn't be done with inline styles IMHO, I'm wondering if centering should be done with flexbox rather then text-align.

@shilman @domyen does the centering of the iframe seem like the right thing to do to you?